src/os/Transaction.h

```diff
@@ -1293,7 +1293,7 @@ class Transaction {
   void dump(ceph::Formatter *f);
   static void generate_test_instances(std::list<Transaction*>& o);
 };
-WRITE_CLASS_ENCODER(Transaction)
+WRITE_CLASS_ENCODER(ceph::os::Transaction)
```

Comment:
nit: for consistency might be worth adding the namespaces also one line below.

Reply:
done

Reply:
:+1: